CONTRIBUTING.md

```diff
@@ -108,8 +108,11 @@ python -m venv venv
 source venv/bin/activate
 # update the version of pip, setuptools, and wheel
 (venv) pip install -U pip setuptools wheel
-# Install the dependencies in your virtual environment
+# Install all the dependencies in your virtual environment.
+#For the zsh shell, you need to specify quotes around the ".[all]" argument but the code below should work fine for bash shell
 (venv) pip install .[all]
```

Comment:
```suggestion
(venv) pip install ".[all]"
```
We can just recommend that everyone put quotes here. Thanks for teaching me something about `zsh`!

Reply:
Okay. Will do that.
You are welcome 😁